### PR TITLE
show open tables

### LINKED ISSUE #183: Support "show open tables"
**What would you like to be added**:

**Why is this needed**:

https://dev.mysql.com/doc/refman/8.0/en/show-open-tables.html

### PR DESCRIPTION
**Pull request by Penglq**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #183 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by dongzl — APPROVED** [review]

File: pkg/mysql/rows.go [hunk 1]
```diff
@@ -350,15 +350,24 @@ func (te TextRow) Scan(dest []proto.Value) error {
...
 			dest[i] = bytesconv.BytesToString(b)
-		case mysql.FieldTypeUint16, mysql.FieldTypeUint8, mysql.FieldTypeInt24, mysql.FieldTypeShort, mysql.FieldTypeTiny, mysql.FieldTypeLong:
-			if dest[i], err = strconv.ParseInt(bytesconv.BytesToString(b), 10, 64); err != nil {
+		case mysql.FieldTypeTiny, mysql.FieldTypeShort, mysql.FieldTypeLong, mysql.FieldTypeInt24,
+			mysql.FieldTypeLongLong, mysql.FieldTypeYear:
+			if te.fields[i].(*Field).flags&mysql.UnsignedFlag > 0 {
+				dest[i], err = strconv.ParseInt(bytesconv.BytesToString(b), 10, 64)
+			} else {
+				dest[i], err = strconv.ParseInt(bytesconv.BytesToString(b), 10, 64)
+			}
```
Comment (jjeffcaii) [inline pkg/mysql/rows.go:361]: Same behavior?

**Review by jjeffcaii — APPROVED** [review]
By the way, can it be merged with `ShowTablesPlan`? I mean, it just one more 'OPEN' keyword.

File: pkg/mysql/rows.go (hunk 1 above)
Comment (Penglq) [reply pkg/mysql/rows.go:361]: Oh, it is wrong. I will change it.

**Comment by Penglq** [comment]
Perform show tables for each database under the tenant

**Review by baerwang — APPROVED** [review]

**Review by georgehao — APPROVED** [review]
LGTM

**Review by AlexStocks — APPROVED** [review]

**Pull request merged**